docs/configuring-metabase/caching.md

```diff
@@ -7,121 +7,107 @@ redirect_from:
-
-Metabase gives you the ability to automatically cache question results that meet a [minimum query duration](#minimum-query-duration).
-
-If your questions share a common model, you can enable [model caching](../data-modeling/models.md#model-caching) instead.
-
-## Enabling global caching
-
-1. Go to **Admin settings** > **Caching** (in the sidebar).
-2. Click the toggle under **Saved Questions**.
-
-Once you've enabled caching, you can choose when and what to cache from your [caching settings](#caching-settings).
-
-By default, questions will get cached once their [average execution time](#average-query-execution-time) meets a [minimum query duration](#minimum-query-duration) of 60 seconds.
+{% include plans-blockquote.html feature="Advanced caching controls" %}
 
-## Caching location
+If your question results don't change frequently, you may want to store the results so that the next time anyone visits the question, Metabase can retrieve the stored results rather than query the database again.
 
-If you're self-hosting Metabase, cached question results will be saved to your [application database](../installation-and-operation/configuring-application-database.md).
+For example, if your data only updates once a day, there's no point in querying the database more than once a day, as they data won't have changed. Returning cached results can be significantly faster, as the database won't have to recompute the results to load your question.
 
-If you're using Metabase Cloud, cached question results will be saved to Metabase's servers in the United States.
+## Cache invalidation policies
 
-## Last updated at
+These policies determines how long cached results will be stored.
 
-Questions that use the cache will display a "last cached at" timestamp in the question's **info** panel.
+- [Duration](#duration-caching-policy)
+- [Schedule](#schedule-caching-policy)
+- [Adaptive](#adaptive-caching-policy)
+- [Don't cache results](#dont-cache-results)
 
-## Getting fresh results
+### Duration caching policy
 
-To override a cached question result, re-run the question using the **refresh** button (counterclockwise arrow).
+Keep the cache for a number of hours. When someone runs a query, Metabase will first check if it's cached the results. If not, it runs the query and caches the results for as long as you set the duration.
 
-## Average query execution time
+### Schedule caching policy
 
-Your Metabase instance keeps track of how long it takes each question to run. The average query execution time is used in your [caching settings](#caching-settings).
+Pick when to regularly invalidate the cache. Metabase will periodically run the query according to the schedule you set and store the results. Other people running queries have no effect on when the results are refreshed; they'll always get the cached results unless they explicitly [clear the cache](#clearing-the-cache).
```

Comment:
Metabase doesn't periodically run queries.

We periodically invalidate the cache.
If it is scheduled to invalidate the cache every 6h starting midnight, we will invalidate the cache (if it exists) at 0, 6, 12, and 18.

If no one runs the query in this interval, there won't be any cache and nothing will be invalidated.
Also, the query will run at least once in the interval (to create the cache)